db-engine/src/main/java/com/splicemachine/db/impl/sql/conn/SessionPropertiesImpl.java

```diff
@@ -50,15 +50,15 @@ public void setProperty(PROPERTYNAME property, Object value) {
 
         switch (property) {
             case USEOLAP:
-                boolean useOlapVal = Boolean.valueOf(valString);
+                boolean useOlapVal = Boolean.parseBoolean(valString);
                 properties[USEOLAP.getId()] = useOlapVal;
                 break;
```

Comment:
Why does the 3.0 code have `case USESPARK` but the other branches don't have it?

Reply:
Because both USEOLAP and USESPARK are defined in SessionProperties 3.0
@martinrupp, why was that the case?